test/validators/movie.test.js

```diff
@@ -44,12 +46,78 @@ describe('movie validator', () => {
         title: 'foo',
         release_year: 12345
       };
-      const result = Joi.validate(payload, MovieValidator);
+      const result = Joi.validate(payload, MovieValidator.payload);
 
       expect(result.error.details[0].path).to.eql('release_year');
       expect(result.error.details[0].type).to.eql('number.max');
     });
 
   });
 
+  describe('release_year_start', () => {
+
+    it('is after 1878', () => {
+      const query = {
+        title: 'poop',
+        release_year_start: 1776
+      };
+      const result = Joi.validate(query, MovieValidator.query);
+
+      expect(result.error.details[0].path).to.eql('release_year_start');
+      expect(result.error.details[0].type).to.eql('number.min');
+    });
+
+    it('is limited to 4 digits', () => {
```

Comment:
nit: A better name might be `is at most 9999` since that's the technical limitation (we're checking a numeric value constraint over a length constraint). Similarly it might be worth testing `10000` over `12345`